feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/components/TimetableItemDetailFloatingMenu.kt

```diff
@@ -71,16 +78,24 @@ fun TimetableItemDetailFloatingActionButtonMenu(
 private fun TimetableItemDetailFloatingActionButtonMenu(
     expanded: Boolean,
     isBookmarked: Boolean,
+    fabBookmarkFlag: Boolean,
     slideUrl: String?,
     videoUrl: String?,
     onExpandedChange: (Boolean) -> Unit,
     onBookmarkClick: (isBookmarked: Boolean) -> Unit,
+    onFabBookmarkFlagChange: (Boolean) -> Unit,
```

Comment:
[nitpick] We probably don’t need to expose `fabBookmarkFlag`. It’s only used for internal UI control (to avoid extra recompositions), so we could keep it local to this composable.